03_launch_mgmt_cluster.sh

```diff
@@ -169,7 +169,7 @@ EOF
   fi
 
   # Copy the generated configmap for ironic deployment
-  if [[ ${BMOBRANCH} == "v0.1.2"  ]]; then # BMORELEASE until v0.1.2 used the old path TODO(mboukhalfa) can be removed after new bmo release
+  if [[ ${BMOBRANCH} == "v0.1.2" || ${BMOBRANCH} == "v0.1.1" ]]; then # BMORELEASE until v0.1.2 used the old path TODO(mboukhalfa) can be removed after new bmo release
```

Comment:
I do not think the TODO comment is still relevant since now we are using BMO releases before we used to always use main

Reply:
I think this TODO was when we wont be using release older than v0.1.2 , so its still relevant?

Reply:
Yes we can keep it then until deprecating 0.1.2